server/router.go

```diff
@@ -51,8 +51,10 @@ func RegisterHandlers(s controllers.Server) {
 
 	// Admin usage endpoints.
 	admin.GET("/usages", s.GetAllActiveUsage)
-	admin.POST("/usages", s.UpdateUsages)
+	// Updates the usage of a particular user and puts an update record in the update table.
+	admin.PUT("/usages", s.UpdateUsages)
 	admin.POST("/usages/:user_name/:resource_name", s.AddUsages)
+	admin.POST(("/updateOP/:update_operation"), s.AddUpdatOperation)
```

Comment:
It's idiomatic to use lower case in URL paths, so it would be nice to set the endpoint to `/update-op/:update_operation` or something like that.